content/en/storage-provider/filecoin-deals/return-on-investment/index.md

```diff
@@ -33,6 +33,7 @@ For more information and context see the following video:
 
 {{< youtube "zboAgawHT-o" >}}
 
+
```

Comment:
Are you trying to add more _padding_ between the YouTube element and the next paragraph? Adding an extra empty line here won't do it. We'd need to edit the CSS.

Reply:
yes indeed, more padding space requried. I feared an extra line wouldn't fix it...

Reply:
Yeah, markdown parsers ignore duplicate empty lines. I've created an issue for this problem: https://github.com/filecoin-project/filecoin-docs/issues/1921

I'll resolve this issue so we can get this PR merged.